craftopia.kvp

```diff
@@ -1,34 +1,74 @@
-App.AdminMethod=STDIO
-App.ApplicationPort1=6587
-App.ApplicationReadyMode=RegexMatch
+Meta.DisplayName=Craftopia
+Meta.Description=
+Meta.OS=Windows, Linux
+Meta.Arch=x86_64
+Meta.Author=Andy Boatman, IceOfWraith
+Meta.URL=https://store.steampowered.com/app/1307550/Craftopia/
+Meta.DisplayImageSource=steam:1307550
+Meta.EndpointURIFormat=steam://connect/{0}
+Meta.ConfigManifest=craftopiaconfig.json
+Meta.MetaConfigManifest=craftopiametaconfig.json
+Meta.ConfigRoot=craftopia.kvp
+Meta.MinAMPVersion=
+Meta.SpecificDockerImage=
+Meta.Prerequsites=[]
+Meta.ConfigReleaseState=NotSpecified
+App.DisplayName=Craftopia
+App.RootDir=./craftopia/
 App.BaseDirectory=./craftopia/1670340/
+App.ExecutableWin=1670340\Craftopia.exe
+App.ExecutableLinux=1670340/Craftopia.x86_64
+App.WorkingDir=1670340
+App.LinuxCommandLineArgs=
+App.WindowsCommandLineArgs=
 App.CommandLineArgs=-batchmode -showlogs
-App.CommandLineParameterDelimiter= 
+App.AppSettings={}
+App.EnvironmentVariables={"LD_LIBRARY_PATH":"./linux64:%LD_LIBRARY_PATH%","SteamAppId":"1670340"}
 App.CommandLineParameterFormat=-{0} "{1}"
-App.DisplayName=Craftopia
-App.EnvironmentVariables={"LD_LIBRARY_PATH": "./linux64:%LD_LIBRARY_PATH%", "SteamAppId": "1670340"}
-App.ExecutableLinux=1670340/Craftopia.x86_64
-App.ExecutableWin=1670340\Craftopia.exe
-App.ExitMethod=String
+App.CommandLineParameterDelimiter= 
+App.ExitMethod=OS_CLOSE
+App.ExitTimeout=30
 App.ExitString=quit
-App.HasReadableConsole=true
-App.HasWritableConsole=true
-App.RapidStartup=false
-App.RootDir=./craftopia/
-App.UpdateSource=Multi
-App.UpdateSources=[{"UpdateStageName": "SteamCMD Download","UpdateSourcePlatform": "All", "UpdateSource": "SteamCMD", "UpdateSourceData": "1670340"}]
-App.WorkingDir=1670340
+App.ExitFile=app_exit.lck
+App.HasWriteableConsole=True
+App.HasReadableConsole=True
+App.SupportsLiveSettingsChanges=False
+App.LiveSettingChangeCommandFormat=set {0} "{1}"
+App.ApplicationIPBinding=0.0.0.0
+App.ApplicationPort1=7778
```

Comment:
Should this be 6587 to match the config file (and the manifest)?

Reply:
Yes, it should. I need to stop drinking while doing this. It's just water, but clearly I was incoherent.